cmake.config.h.in

```diff
@@ -0,0 +1,429 @@
+/* config.h.  Generated from config.h.in by configure.  */
+/* config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
Can this file be automatically generated, e.g. by adding a few lines to `bootstrap.sh`?